core/src/commonMain/kotlin/exception/EntityNotFoundException.kt

```diff
@@ -104,5 +105,8 @@ public class EntityNotFoundException : Exception {
 
         public inline fun autoModerationRuleNotFound(guildId: Snowflake, ruleId: Snowflake): Nothing =
             guildEntityNotFound("Auto Moderation Rule", guildId, ruleId)
+
+        public inline fun entitlementNotFound(applicationId: Snowflake, entitlementId: Snowflake): Nothing =
+            throw EntityNotFoundException("Entitlement with id $entitlementId in application $applicationId was not found.")
```

Comment:
```suggestion
            throw EntityNotFoundException("Entitlement with id $entitlementId for application $applicationId was not found.")
```
not a native speaker, but "for" sound like the correct word to me here